### ISSUE TITLE
Website has issues on my company network

### ISSUE THREAD

**Issue by Irubataru** [FastEndpoints/FastEndpoints#251]
Sorry, I don't think this is the right place to put it, but the [Documentation repo](https://github.com/FastEndpoints/FastEndpoints) doesn't have an issue tracker.

It's a bit specific, but I have never had any issues with websites on the company network before so I'll ask here and maybe you can fix it, or maybe you can tell me what to tell IT so they can fix whatever is wrong on their end.

Anyway, when I try to access the [site](https://fast-endpoints.com/) (I have tried brave, firefox, and qutebrowser on Linux, and edge on Windows) I get an error `500: e is not iterable`. When I check the console I have an error:

```
index.svelte-60282696.js:3
       GET https://api.github.com/repos/dj-nitehawk/FastEndpoints/contributors 403
$t @ index.svelte-60282696.js:3
Xr @ index.svelte-60282696.js:12
se @ start-490252bd.js:1
_hydrate @ start-490252bd.js:1
await in _hydrate (async)
zt @ start-490252bd.js:1
(anonymous) @ (index):279
```

Does it perhaps have something to do with github api rate limit?

**Comment by dj-nitehawk** [comment]
thanks for reporting. we're looking in to it: https://discord.com/channels/933662816458645504/999224616595750922/1022068461339422731

**Comment by dj-nitehawk** [comment]
@Irubataru do you still have this issue?
i'll close this issue for now since it hasn't had much activity.

**Comment by Irubataru** [comment]
Yeah, it still doesn't work for me. I assume my company has a maxed out api request limit at all times.
